root/etc/cont-init.d/98-dashboard-config

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/with-contenv bash
+
+echo "Applying the SWAG dashboard mod..."
+
+apk add --no-cache --upgrade goaccess
+cp -f /dashboard/dashboard.subdomain.conf.sample /config/nginx/proxy-confs/
```

Comment:
if they install the mod, they already show they want this, i think you should copy it in without .sample; any other thoughts?